src/main/java/com/cloudbees/hudson/plugins/folder/computed/ComputedFolder.java

```diff
@@ -342,7 +342,9 @@ protected final String getSuccessfulDestination() {
         return triggers.toMap();
     }
 
+    @Restricted(NoExternalUse.class) // currently used only by jelly / stapler
     public List<TriggerDescriptor> getTriggerDescriptors() {
+        // TODO remove this once core has support for DescriptorVisibilityFilter on Trigger.for_(Item)
```

Comment:
yes, we would need to add a descriptor visibility filter as part of removing this method